### ISSUE TITLE
MotherVR controls display projected at uncomfortable depth

### ISSUE THREAD

**Issue by Jaerin** [Nibre/MotherVR#219]
I've been finding that reading terminals and the like can be quite uncomfortable.  The problem seems that when you sit down at a terminal the screen depth of that terminal is pretty close to your eyes.  So close that the controls bar at the bottom that MotherVR projects into the game is significantly further away than the plane of the terminal screen.  This causes me to feel like I am crossing my eyes to focus on the terminal plane to read the screen while also having a cross-eyed stereoscopic vision of the menu bar.  An option to allow you to adjust the nominal depth of the control bar or the option to turn it off entirely would help greatly.

**Comment by JaesenMoreaux** [comment]
Definitely. I'm glad this mod lets us move away from the terminal, unlike the way you were forced into the screen without this mod, but the hovering menu seems to be at a different depth. Need the ability to adjust these hovering HUD items to different parts of the screen or at different depths.
